components/sendinblue/actions/add-or-update-contact/add-or-update-contact.mjs

```diff
@@ -0,0 +1,101 @@
+import sendinBlueApp from "../../sendinblue.app.mjs";
+
+// legacy_hash_id: a_0Mi7n5
+export default {
+  key: "sendinblue-add-or-update-contact",
+  name: "Add or Updated a contact",
+  description: "Add or Updated a contact and list",
+  version: "0.0.79",
```

Comment:
You'll want to update these to read `0.0.1`. See https://pipedream.com/docs/components/guidelines/#versioning